From: jrpelegrina <juapesai@hotmail.com>
Date: Fri, 11 Jan 2019 13:23:09 +0100
Subject: Apply Lliurex patch

---
 moodle/admin/index.php | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/moodle/admin/index.php b/moodle/admin/index.php
index e04c319..082fb12 100644
--- a/moodle/admin/index.php
+++ b/moodle/admin/index.php
@@ -733,9 +733,9 @@ if (during_initial_install()) {
 
     if ($adminuser->password === 'adminsetuppending') {
         // prevent installation hijacking
-        if ($adminuser->lastip !== getremoteaddr()) {
-            print_error('installhijacked', 'admin');
-        }
+        //if ($adminuser->lastip !== getremoteaddr()) {
+        //    print_error('installhijacked', 'admin');
+       // }
         // login user and let him set password and admin details
         $adminuser->newadminuser = 1;
         complete_user_login($adminuser);
